src/Microsoft.ML.StandardLearners/Standard/Online/AveragedLinear.cs

```diff
@@ -18,36 +18,71 @@ namespace Microsoft.ML.Trainers.Online
 {
     public abstract class AveragedLinearArguments : OnlineLinearArguments
     {
+        /// <summary>
+        /// <a href="tmpurl_lr">Learning rate</a>
+        /// </summary>
         [Argument(ArgumentType.AtMostOnce, HelpText = "Learning rate", ShortName = "lr", SortOrder = 50)]
         [TGUI(Label = "Learning rate", SuggestedSweeps = "0.01,0.1,0.5,1.0")]
         [TlcModule.SweepableDiscreteParam("LearningRate", new object[] { 0.01, 0.1, 0.5, 1.0 })]
         public float LearningRate = AveragedDefaultArgs.LearningRate;
 
+        /// <summary>
+        /// <see langword="true" /> to decrease the <a href="tmpurl_lr">learning rate</a> as iterations progress; otherwise, <see langword="false" />.
```

Comment:
><a href="tmpurl_lr"> [](start = 52, length = 20)

i think the <a> can live above, and here can be a cref.

Reply:
yes. cref is better.

---
In reply to: [255267344](https://github.com/dotnet/machinelearning/pull/2483#discussion_r255267344) [](ancestors = 255267344)